apps/playwright/tests/support-page.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, test } from "~/fixtures";
+
+test.describe("Support page", () => {
+  test.describe("Support page", () => {
+    test("has heading", async ({ page }) => {
+      await page.goto("/en/support");
+      await expect(
+        page.getByRole("heading", { name: "Contact and support", level: 1 }),
+      ).toBeVisible();
+    });
+    test("has no accessibility violations", async ({
+      appPage,
+      makeAxeBuilder,
+    }) => {
+      await appPage.page.goto("/en/support");
+      const accessibilityScanResults = await makeAxeBuilder().analyze();
+      expect(accessibilityScanResults.violations).toEqual([]);
+    });
+  });
+  test.describe("Support form", () => {
+    test("send POST request to existing API endpoint", async ({ request }) => {
+      const newTicket = await request.post("/api/support/tickets", {
+        data: {
+          body: {
+            name: "Test Person",
+            email: "test@test.tld",
+            details: "Test comments to send.",
+            subject: "question",
+          },
+        },
+      });
+      expect(newTicket.status()).toBe(500); // 500 status if freshdesk is not locally configured.
+    });
```

Comment:
The conditional from the cypress test where it runs an additional assertion if freshdesk is configured is missing. Do you want to add that it in or is it not worth it?

Reply:
The test itself isn't great since a 500 error code response doesn't tell much about the endpoint's validity other than getting a response, I guess. The Freshdesk config vars are always empty in our tests, so not sure adding the conditional adds any value.